source/common/config/utility.h

```diff
@@ -209,13 +211,50 @@ class Utility {
     return *factory;
   }
 
+  template <class Factory> static Factory& getAndCheckFactoryByType(absl::string_view type) {
+    if (type.empty()) {
+      throw EnvoyException("Provided type for static registration lookup was empty.");
+    }
+
+    Factory* factory = Registry::FactoryRegistry<Factory>::getFactoryByType(type);
+
```

Comment:
Nit: prefer more compact spacing.

Reply:
Yeah, this is just a copy-paste of getFactoryByName which has too much spacing. Compacting.